src/main/java/com/lorepo/icplayer/client/module/text/GapInfo.java

```diff
@@ -103,13 +103,12 @@ public boolean isCorrect(String text) {
 		return correct;
 	}
 
-	public boolean isTextOnlyPlaceholder(String text, boolean ignorePlaceholderWhenChecking) {
-	    text = getCleanedText(text);
-	    if (textEqualsPlaceholder(text) && ignorePlaceholderWhenChecking) {
-            return true;
-		}
-		return false;
-	}
+    public boolean isValueCheckable(boolean ignorePlaceholderWhenChecking, boolean hasGapBeenAccessed) {
+        if (ignorePlaceholderWhenChecking && !hasGapBeenAccessed) {
```

Comment:
uwaga estetyczna:
return !(ignorePlaceholderWhenChecking && !hasGapBeenAccessed);

Reply:
Tutaj zostawiam celowo - imo dużo czytelniejsze jest w aktualny sposób - jeżeli ignorePlaceholderWhenChecking jest włączony a gapa nie została accessed to funkcja isValueCheckable zwraca False. Przy podwójnej negacji przeważnie czytelność trochę się rozmywa.